# 0: absent, 1: attemp to sycn, 2: locked, 3: error
uint8 pps_state

# "V" invalid, "A" valid
uint8 location_valid

uint32 gps_timestamp
